Orange/distance/__init__.py

```diff
@@ -72,13 +76,15 @@ def __call__(self, e1, e2=None, axis=1, impute=False):
             x1 = x1.T
             if x2 is not None:
                 x2 = x2.T
-        dist = skl_metrics.pairwise.pairwise_distances(x1, x2, metric=self.metric)
+        dist = skl_metrics.pairwise.pairwise_distances(x1, x2,
+                                                       metric=self.metric)
```

Comment:
Although this is PEP8-compliant, arguments aligned with `(` in such places do not work well.

```
        dist = skl_metrics.pairwise.pairwise_distances(
            x1, x2, metric=self.metric)
```

is better. See a more evident case below.